runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -6492,6 +6492,274 @@ static TR::Register* inlineIntrinsicIndexOf(TR::Node* node, TR::CodeGenerator* c
    return resultReg;
    }
 
+/**
+ * @brief Generates inlined instructions equivalent to java/lang/StringLatin1.inflate(byte[] src, int srcOff, char[] dst, int dstOff, int len)
+ *
+ * @param node: node
+ * @param cg: Code Generator
+ * @returns register
+ */
+static TR::Register *inlineStringLatin1Inflate(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   /*
+    *    cbz      lengthReg, LDONE
+    *    add      srcAddrReg, srcReg, srcOffsetReg
+    *    add      dstAddrReg, dstReg, dstOffsetReg, lsl #1
+    *    cmp      lengthReg, #2
+    *    b.cs     GEQ2Bytes
+    *    ldrb     dataReg, [srcAddrReg]
+    *    strh     dataReg, [dstAddrReg]
+    *    b        LDONE
+    * GEQ2Bytes:
+    *    add      srcEndReg, srcAddrReg, lengthReg
+    *    add      dstEndReg,  dstAddrReg, lengthReg, lsl #1
```

Comment:
Please remove one of the space characters between `dstEndReg,` and `dstAddrReg`.

Reply:
Removed the space character.